derive-new-test/Cargo.lock

```diff
@@ -1,38 +0,0 @@
-[root]
```

Comment:
(I also tried following [the official advice](http://doc.crates.io/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries) which states that for libraries Cargo.lock should not be checked in into version control; not sure how relevant that is...)